### DISCUSSION TITLE
REFRPROPdll enthalpy does not match GUI

### DISCUSSION THREAD

**Discussion by charlespw** [usnistgov/REFPROP-issues#621] [Q&A]
I am seeing a discrepancy between the outputs of REFPROPdll using ctREFPROP vs. the gui. Using the following code, I get an output of 1002.68 for Enthalpy. Using the GUI, I get value of 1255.8. What might I be doing wrong here?
```
from ctREFPROP.ctREFPROP import REFPROPFunctionLibrary

RP = REFPROPFunctionLibrary(os.environ['RPPREFIX'])
RP.SETPATHdll(os.environ['RPPREFIX'])

#create unit enum, set units to SI with C
SI_WITH_C = RP.GETENUMdll(0,"SI WITH C").iEnum

def RP2d(hFld,hIn1,a,hIn2,b,hOut,z):
    r = RP.REFPROPdll(hFld,hIn1+hIn2,hOut,SI_WITH_C,0, 1,a, b, z)
    if r.ierr != 0:
        print(r.ierr,r.herr) #print error flag (0 indicates no error), herror string (if no error, string contained in hUnits will be sent), and output (c)
    else:
        return r.Output[0],r.q

T_outlet = 50
hFld='NITROGEN;CARBON DIOXIDE;WATER'
zFld = [0.5,0.1,0.4]
H_expanded = RP2d(hFld,'P',0.101325,'T',T_outlet,'H',zFld)[0]
```
<img width="517" alt="Screenshot 2023-10-31 151337" src="https://github.com/usnistgov/REFPROP-issues/assets/20848572/406926ba-181f-413b-835b-6d4677384768">

**Comment by ianhbell** [comment]
The GUI and the DLL use different default reference states. Check instead an enthalpy difference between two T,p, points; those should agree. Some variant of this question is the most popular FAQ
